shuup/admin/modules/orders/views/addresses.py

```diff
@@ -26,6 +26,10 @@ class OrderAddressEditView(UpdateView):
     context_object_name = "order"
     template_name = "shuup/admin/orders/_address_edit.jinja"
 
+    def get_queryset(self):
+        shop_ids = Shop.objects.get_for_user(self.request.user).values_list("id", flat=True)
```

Comment:
maybe use get_shop here

Reply:
For these I do not see necessary for current shop to be active. I just want to ensure that not wrong people can access these views.